drydock/templates/kustomized/tutor16/base/apps/openedx/settings/partials/common_all.py

```diff
@@ -0,0 +1,223 @@
+####### Settings common to LMS and CMS
+import json
+import os
+
+from xmodule.modulestore.modulestore_settings import update_module_store_settings
+
+# Mongodb connection parameters: simply modify `mongodb_parameters` to affect all connections to MongoDb.
+mongodb_parameters = {
+    "db": "{{ MONGODB_DATABASE }}",
+    "host": "{{ MONGODB_HOST }}",
+    "port": {{ MONGODB_PORT }},
+    "user": {% if MONGODB_USERNAME %}"{{ MONGODB_USERNAME }}"{% else %}None{% endif %},
+    "password": {% if MONGODB_PASSWORD %}"{{ MONGODB_PASSWORD }}"{% else %}None{% endif %},
+    # Connection/Authentication
+    "ssl": {{ MONGODB_USE_SSL }},
+    "authSource": "{{ MONGODB_AUTH_SOURCE }}",
```

Comment:
```suggestion
    "authsource": "{{ MONGODB_AUTH_SOURCE }}",
```
From https://github.com/overhangio/tutor/pull/841